CHANGELOG.md

```diff
@@ -32,6 +32,8 @@
 
   ([PR #58](https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/58))
 
+## 0.5.0
+
```

Comment:
This release also includes the fix above 😁

Reply:
OK, I would have never spotted that. Please can we do something to make the actual changes distinct from the template?

Reply:
Good idea... how? 🤔

Reply:
I've added a commit.